src/metax_api/tests/api/rest/base/views/directories/read.py

```diff
@@ -367,12 +368,14 @@ def test_read_directory_for_not_catalog_record_recursively(self):
         """
         Test query parameters 'not_cr_identifier' with 'recursive'.
         """
-        response = self.client.get('/rest/directories/1/files?recursive&not_cr_identifier=%s&depth=*'
+        cr_recursive = self.client.get('/rest/directories/1/files?recursive&depth=*').data
```

Comment:
``cr_recursive`` The API returns files, not catalog records?

Reply:
same as before, mixing terminology